android/quest/src/test/java/org/smartregister/fhircore/quest/app/fakes/Faker.kt

```diff
@@ -35,6 +44,34 @@ object Faker {
       refreshToken = "USrAgmSf5MJ8N_RLQODa7rZ3zNs1Sj1GkSIsTsb4n-Y"
     )
 
+  private const val APP_DEBUG = "app/debug"
+
+  fun buildTestConfigurationRegistry(): ConfigurationRegistry {
+
+    val fhirResourceService = mockk<FhirResourceService>()
+    val fhirResourceDataSource = spyk(FhirResourceDataSource(fhirResourceService))
+    coEvery { fhirResourceService.getResource(any()) } returns Bundle()
+
+    val configurationRegistry =
+      spyk(
+        ConfigurationRegistry(
+          fhirResourceDataSource = fhirResourceDataSource,
+          sharedPreferencesHelper = mockk(),
+          dispatcherProvider = mockk(),
+          repository = mockk()
+        )
+      )
+
+    runBlocking {
+      configurationRegistry.loadConfigurations(
+        appId = APP_DEBUG,
+        context = InstrumentationRegistry.getInstrumentation().targetContext
+      ) {}
+    }
+
+    return configurationRegistry
+  }
```

Comment:
I do not think this was necessary for quest app because we can just inject and configuration registry and call the load configs method in the setup function. It is still okay to have this anyways. But I do think we need to provided the configs in the `assets` for the `test`  because we can't rely on the `main` package  `assets` which are bound to change multiple times thus affecting the tests.

Reply:
Ah I see, yes I can add them.

Actually we couldn't use Inject in this case, because at some point it will call `fhirResourceService.getResource()` which will fail the test case because of network stuff error.

The idea is to keep line 51 to 53.

Reply:
Okay